---
layout: post
title:  "Population dynamics of rDNA copy number in yeast and humans"
comments: true
category: genomics
description: "<b>Diksha Sharma, Sylvie Hermann Le Denmat, Nicholas Matzke, Lauri Saag, Katherine Hannan, Ross Hannan, Herwati Sudoyo, Murray Cox, Justin Sullivan, Austen Ganley</b><br/>In most eukaryotes the ribosomal RNA genes (rDNA) ..."
videoID: https://cloudstor.aarnet.edu.au/plus/s/EPwFGrxa6mD7P0A/download
optimized_image: https://cloudstor.aarnet.edu.au/plus/s/XJbJLtDbihiB7kB/download
tags:
 - sequence variation
 - phylodynamics
 - rDNA
---
{% include cloudstorplayer.html id=page.videoID %}
<u>Diksha Sharma</u><sup>0</sup>, Sylvie Hermann Le Denmat<sup>0</sup>, Nicholas Matzke<sup>0</sup>, Lauri Saag<sup>1</sup>, Katherine Hannan<sup>2</sup>, Ross Hannan<sup>3</sup>, Herwati Sudoyo<sup>4</sup>, Murray Cox<sup>5</sup>, Justin Sullivan<sup>6</sup>, Austen Ganley<sup>0</sup><br/>
\(0\) School of Biological Sciences, University of Auckland, Auckland, New Zealand<br/>
\(1\) Institute of Genomics, University of Tartu, Tartu, Tartumaa, 51010, Estonia<br/>
\(2\) ACRF Department of Cancer Biology and Therapeutics, The John Curtin School of Medical Research, ACT 2601, Australia<br/>
\(3\) Peter MacCallum Cancer Centre; Australian National University; University of Melbourne ; Monash University Australia<br/>
\(4\) Eijkman Institute for Molecular Biology ; University of Indonesia ; University of Sydney<br/>
\(5\) Massey University,  New Zealand<br/>
\(6\) Liggins Institute, University of Auckland, Auckland, New Zealand

Find me on Wed Nov 25th, 1:30-2:50pm AEDT [in Remo, table 138](https://live.remo.co/e/abacbs2020-day-2/register)

<b>Abstract</b><br/>
In most eukaryotes the ribosomal RNA genes \(rDNA\) are organized as tandem repeat arrays, with each species having a characteristic number of copies. Despite this, there is a high degree of variability in copy number between individuals, and the copy number is thought to return to the “homeostatic” number when perturbed. However, it is unclear whether homeostatic rDNA copy number is a species-level property, or whether different populations have different homeostatic rDNA copy numbers. To address this question, we developed a novel bioinformatics approach to measure rDNA copy number from whole genome sequence data using the most frequent \(modal\) coverage. We validated this method with Saccharomyces cerevisiae strains having known, different rDNA copy numbers, and show this approach is robust and reliable, even with low coverage datasets. We then applied our pipeline to investigate variation in rDNA copy number between different S. cerevisiae and human populations. Our results using 1002 S. cerevisiae Genome project data suggest that different populations have different homeostatic rDNA copy numbers. We validated this result using a molecular biology approach, with yeast isolates from different populations showing no recovery to a common rDNA copy number. In contrast, investigation of 430 individuals coming from 14 populations from around the world showed no evidence for population-level differences in rDNA copy number, except for the Papuan population that might harbor a higher homeostatic rDNA copy number than other populations. Together, this work establishes a robust and a simple platform to determine rDNA copy number using whole genome sequence data. This pipeline provides evidence for population-level differences in S. cerevisiae, but little evidence for such differences in most human populations. This discrepancy might result from S. cerevisiae populations have deeper genetic divergence, and/or differences in the population dynamics of these species.
