---
layout: post
title:  "Hypothesis generation in the age of cell atlases"
comments: true
category: single_cell
description: "<b>Fabio Zanini, Julie Thoms, Bojk Berghuis, Racquel Domingo-Gonzales, Zhiyuan Yao, Shirit Einav, Cristina Alvira, David Cornfield, Stephen Quake, John Pimanda</b><br/>While many laboratories are driven by computationa..."
videoID: https://cloudstor.aarnet.edu.au/plus/s/YkbySfwIKBtnrcA/download
optimized_image: https://cloudstor.aarnet.edu.au/plus/s/vImjVfqkjspCbh0/download
invited_talk: 1
tags:
 - single cell
 - viruses
 - transcriptomics
 - development
 - clustering
---
{% include cloudstorplayer.html id=page.videoID %}
[<u>Fabio Zanini</u>](http://fabilab.org)<sup>0</sup>, Julie Thoms<sup>0</sup>, Bojk Berghuis<sup>1</sup>, Racquel Domingo-Gonzales<sup>1</sup>, Zhiyuan Yao<sup>1</sup>, Shirit Einav<sup>1</sup>, Cristina Alvira<sup>1</sup>, David Cornfield<sup>1</sup>, Stephen Quake<sup>2</sup>, John Pimanda<sup>0</sup><br/>
\(0\) UNSW Sydney<br/>
\(1\) Stanford University<br/>
\(2\) Stanford University, Chan Zuckerberg Biohub


<b>Abstract</b><br/>
While many laboratories are driven by computational method development or hypothesis-driven biomedical experimentation, we \(http://fabilab.org\) focus on data-driven hypothesis generation, especially single cell sequencing of viral infections, development, and cancer. We recently released northstar \(Zanini et al. Scientific Reports 10, 15251 \(2020\)\), a new algorithmic approach and software package to cluster/classify single cell transcriptomes guided by but not limited by a cell atlas. Northstar can annotate tens of thousands of cells from tumor samples into known cell types and unknown clusters within seconds on a laptop. We subsequently combined northstar with RNA velocity, ATAC-Seq, ChIP-Seq, knockdown, and overexpression to study gene regulation in acute myeloid leukemia \(AML\) and discovered an unknown level of heterogeneity within cancer cells. Single cell gene expression, chromatin state at enhancers, transcription factor binding motifs, and perturbation experiments highlighted an internal regulatory network that is modulated by the triad of transcription factors GATA2, TAL1, and ERG and shapes the phenotype of leukemic cells along trajectories that are distinct from but closely related to healthy haematopoiesis, suggesting new therapeutic approaches for leukemia. We will also illustrate our efforts on two distinct lines of research: \(1\) comparative single cell omics of viral infections and \(2\) constructing a cell atlas of the neonatal mammalian lung.
