---
layout: post
title:  "Anthroponotic transmission and adaptive introgression underlies cryptic population structure of Cryptosporidium hominis in Africa "
comments: true
category: genomics
description: "<b>Swapnil Tichkule, Aaron Jex, Cock Van Oosterhout, Simone Cacciò</b><br/>INTRODUCTION: Cryptosporidiosis is a major cause o..."
videoID: asdf
optimized_image: assets/img/x2yM7LcXdCSi0bm_title.jpg
tags:
 - Cryptosporidium hominis
 - whole genome sequencing
 - comparative genomics
 - population structure
 - recombination
 - genetic introgression
---
{% include cloudstorplayer.html id=page.videoID %}
<u>Swapnil Tichkule</u><sup>0</sup>, Aaron Jex<sup>0</sup>, Cock Van Oosterhout<sup>1</sup>, Simone Cacciò<sup>2</sup><br/>
\(0\) Walter &amp; Eliza Hall Institute<br/>
\(1\) University of East Anglia<br/>
\(2\) Istituto Superiore di Sanità

Find me during our live conference, [in Remo Room 3, table 10](https://remo.co)

<b>Abstract</b><br/>
INTRODUCTION: Cryptosporidiosis is a major cause of diarrhoeal illness among African children, and it is associated with increased childhood mortality, malnutrition, cognitive development and growth retardation. Cryptosporidium hominis \(C. hominis\) is the dominant pathogen for this disease in Africa. Genotyping at the glycoprotein 60 \(gp60\) gene has revealed a complex distribution of different subtypes across Africa. However, a comprehensive exploration of the metapopulation structure and evolution based on whole genome data has yet to be performed. <br/><br/>METHODS: In this study, we sequenced and analyzed the genomes of 26 C. hominis isolates, representing different gp60 subtypes, collected at rural sites in Gabon, Ghana, Madagascar and Tanzania. These whole genomes were subjected to variant calling with GATK pipeline, followed by PCA, Phylogeny, STRUCTURE and network based analysis to infer cryptic population structure. HybridCheck and RDP4 programs were employed to investigate introgression and recombination among the isolates, respectively. Polymorphic genes were then evaluated with population genetic metrices and tested their significance towards transmission and evolution. <br/><br/>RESULTS: Phylogenetic and cluster analyses based on Single Nucleotide Polymorphisms showed that isolates predominantly clustered by their country of origin, irrespective of their gp60 subtype. We found a significant isolation-by-distance signature that shows the importance of local transmission, but we also detected evidence of hybridization between isolates of different geographic regions. We identified 37 outlier genes with exceptionally high nucleotide diversity, and this group is significantly enriched for genes encoding extracellular proteins and signal peptides. Furthermore, these genes are found more often than expected in recombinant regions, and they show a distinct signature of positive- or balancing selection.<br/><br/>CONCLUSION: Our study showed that: 1\) the metapopulation structure of C. hominis can only be accurately captured by whole genome analyses; 2\) local anthroponotic transmission underpins the spread of this pathogen in Africa; 3\) hybridization occurs between distinct geographical lineages; and 4\) genetic introgression provides novel substrate for positive- or balancing selection in genes involved in host-parasite coevolution.<br/>
