---
layout: post
title:  "Accurate detection of cell free microbial DNA using a contaminant-controlled analysis framework"
comments: true
category: metagenomics
description: "<b>Enrique Zozaya-Valdés, Stephen Q. Wong, Jeanette Raleigh, Athena Hatzimihalis, Sarah Ftouni, Anthony T. Papenfuss, Shahneen Sandhu, Mark A. Dawson, Sarah-Jane Dawson</b><br/>The human microbiome plays an important role in ca..."
videoID: https://cloudstor.aarnet.edu.au/plus/s/OC7MU4XxodVud99/download
optimized_image: https://cloudstor.aarnet.edu.au/plus/s/NzswBAXWM4OUZ4I/download
tags:
 - metagenomics
 - microbiome
 - cell free DNA
 - cancer
 - disease
---
{% include cloudstorplayer.html id=page.videoID %}
<u>Enrique Zozaya-Valdés</u><sup>0</sup>, Stephen Q. Wong<sup>0</sup>, Jeanette Raleigh<sup>0</sup>, Athena Hatzimihalis<sup>0</sup>, Sarah Ftouni<sup>0</sup>, Anthony T. Papenfuss<sup>0</sup>, Shahneen Sandhu<sup>0</sup>, Mark A. Dawson<sup>0</sup>, Sarah-Jane Dawson<sup>0</sup><br/>
\(0\) Peter MacCallum Cancer Centre

Find me during our live conference, [in Remo, table 6](https://remo.co)

<b>Abstract</b><br/>
The human microbiome plays an important role in cancer. Accumulating evidence indicates that commensal microbiome-derived DNA may be represented in minute quantities in the cell-free DNA of human blood and could possibly be harnessed as a new cancer biomarker. However, there has been limited use of rigorous experimental controls to account for contamination, which invariably affects low-biomass microbiome studies. Here, we applied a combination of 16S-rRNA-gene sequencing and droplet digital PCR to determine if the specific detection of cell free microbial DNA \(cfmDNA\) was possible in 69 metastatic melanoma patients. Compared to matched stool and saliva samples, the absolute concentration of cfmDNA was low \(mean of 2,719 gene copies/ml of plasma\) but was significantly above the levels detected from negative controls \(mean of 1,829 gene copies/ml\). The microbial community of plasma was strongly influenced by laboratory and reagent contaminants introduced during the DNA extraction and sequencing processes. Through the application of an in silico decontamination strategy including the filtering and removal of Amplicon Sequence Variants \(ASVs\) with batch dependant abundances and those with a higher prevalence in negative controls, 31 high confidence plasma ASVs were identified. These included known gut commensal bacteria \(e.g. Faecalibacterium, Bacteroides and Ruminococcus\) but also other uncharacterised ASVs. Together, these observations indicate that plasma can harbour a low yet detectable level of cfmDNA. These results highlight the importance of accounting for contamination and provide an analytical decontamination framework to allow the accurate detection of cfmDNA for future biomarker studies in cancer and other diseases. 
