---
layout: post
title:  "Implementing an integrated analysis to identify and validate circular RNAs using patient-derived neuronal stem cells."
comments: true
category: transcriptomics
description: "<b>Oak Hatzimanolis, Alan Mackay-Sim, Alexandre Cristino</b><br/>In recent years it has been determined that a larg..."
videoID: https://cloudstor.aarnet.edu.au/plus/s/sbjWIqBI3PFccHO/download
optimized_image: https://cloudstor.aarnet.edu.au/plus/s/Z5xzbjFtSEw53M2/download
session_talk: 1
tags:
 - transcriptomics
 - circular RNAs
 - disease
---
{% include cloudstorplayer.html id=page.videoID %}
<u>Oak Hatzimanolis</u><sup>0</sup>, Alan Mackay-Sim<sup>0</sup>, Alexandre Cristino<sup>0</sup><br/>
\(0\) Griffith University

Find me on Wed Nov 25th, 1:30-2:50pm AEDT [in Remo, table 34](https://live.remo.co/e/abacbs2020-day-2/register)

<b>Abstract</b><br/>
In recent years it has been determined that a large proportion of non-coding RNAs have biochemical functionality \(~80%\), contrary to pre-existing consensus a few decades ago. Circular RNAs \(circRNA\) are an exciting addition to this class of RNA, and since 2012 a significant amount of research has accumulated around these biomolecules. They are regulatory non-coding RNAs playing a critical role in many cellular pathways with implications in cancers and neurological diseases. The most common function attributed to circRNAs is their ability to act as a microRNA sponge. Advances in the fields of computational biology and molecular biology have allowed for widespread study of this class of RNA, with several specific detection tools that have arisen since the start of the last decade. This study aims to take advantage of these recent advances in transcriptomics and bioinformatics to identify circRNAs in schizophrenia patients and healthy controls from total RNA sequencing data from olfactory neuronal stem \(ONS\) cells.  We will use these patient derived ONS cells as a model for neurological diseases as they have previously been used in several other multi-omics studies describing molecular and cellular differences between schizophrenia individuals and healthy controls.  A large amount of circRNA was identified from the total RNA sequencing data and a subsequent differential expression analysis was conducted on these molecules. Over 500 circRNAs were differentially expressed between schizophrenia and controls, with CADM3 being the most downregulated in Schizophrenia and GPR133 being the most upregulated.  Potential MicroRNA regulation was also examined, wherein we found potential sponging occurring from microRNAs found with predicted strong binding to the identified circRNAs. Interestingly we also found microRNAs implicated in schizophrenia such as miR-137 and miR-2682 which were found to putatively bind to some of the identified circRNAs. Several circRNAs such as DOCK1, COL4A2, PTPN13, PRKCA, FAT4, LAMA3 and ITGA3 had a tremendous amount of potential microRNA binding sites suggesting they play an important role as decoyer for microRNAs which effectively decrease the functional amount of microRNA in the cells. Future study into circRNAs could prove to be fruitful in discovering links between diseases states and genetic dysregulation, as well has helping to build the framework for observing neurological diseases as the result of the shared dysregulation of multiple pathways such as the interaction networks of circRNA-miRNA-mRNA-protein.  
