---
layout: post
title:  "Comprehensive characterization of single cell full-length isoforms in human and mouse with long-read sequencing"
comments: true
category: long_reads
description: "<b>Luyi Tian, Jafar Jabbari, Matthew Ritchie</b><br/>Alternative splicing shapes the phenotype of cells..."
videoID: asdf
optimized_image: assets/img/x2yM7LcXdCSi0bm_title.jpg
session_talk: 1
tags:
 - single cell genomic
 -  long-read sequencing
 -  single cell multi-omics
 -  transcriptomic
---
{% include cloudstorplayer.html id=page.videoID %}
<u>Luyi Tian</u><sup>0</sup>, Jafar Jabbari<sup>1</sup>, Matthew Ritchie<sup>0</sup><br/>
\(0\) Walter and Eliza Hall Institute of Medical Research<br/>
\(1\) AGRF

Find me during our live conference, [in Remo Room 2, table 29](https://remo.co)

<b>Abstract</b><br/>
Alternative splicing shapes the phenotype of cells in development and disease. Longread RNA-sequencing recovers full-length transcripts but has limited throughput at the single-cell level. Here we developed single-cell full-length transcript sequencing by sampling \(FLT-seq\), together with the computational pipeline FLAMES to overcome these issues and perform isoform discovery and quantification for single cells. With FLT-seq and FLAMES, we performed the first comprehensive characterization of the full-length isoform landscape in single cells of different cell types and species, identified thousands of unannotated isoforms. We found conserved functional modules that were enriched for alternative transcript usage in different cell populations, including ribosome biogenesis and mRNA splicing. Analysis at the transcript-level allowed data integration with scATAC-seq on individual promoters, improved correlation with protein expression data and linked mutations known to confer drug resistance to transcriptome heterogeneity. Our methods reveal previously unseen isoform complexity and provide a better framework for multi-omics data integration.
