---
layout: post
title:  "qc3C â€“ reference-free quality assessment for Hi-C sequencing data."
comments: true
category: other
description: "<b>Matthew DeMaere, Aaron Darling</b><br/>Hi-C is a sample preparation method that enables h..."
videoID: asdf
optimized_image: assets/img/x2yM7LcXdCSi0bm_title.jpg
tags:
 - dna sequencing
 - hi-c sequencing
 - hi-c quality control
 - next generation sequencing
 - k-mer analysis
 - reference free quality control
---
{% include cloudstorplayer.html id=page.videoID %}
[<u>Matthew DeMaere</u>](https://github.com/cerebis)<sup>0</sup>, [Aaron Darling](http://darlinglab.org/)<sup>0</sup><br/>
\(0\) ithree institute, UTS

Find me during our live conference, [in Remo Room 3, table 2](https://remo.co)

<b>Abstract</b><br/>
Hi-C is a sample preparation method that enables high-throughput sequencing to capture genome-wide proximity interactions between DNA molecules. The technique has been successfully applied to solve challenging problems such as 3D structural analysis of chromatin, scaffolding of large genome assemblies and more recently the accurate resolution of metagenome-assembled genomes \(MAGs\). <br/> <br/>Despite continued refinements, however, Hi-C library preparation remains a costly and complex laboratory protocol. QC options for Hi-C libraries are limited, with current wet-lab protocols only giving a very crude assay for the re-digest of ligation junctions in a Hi-C library. This QC approach does not provide a reliable estimate of the fraction of library templates containing a Hi-C junction, nor is it available to all Hi-C protocols. QC via sequencing is another possible approach, but current tools require a reference genome to estimate quality metrics for the Hi-C library.<br/> <br/>We propose a new, reference-free approach for Hi-C library quality assessment that requires only a small amount of sequencing data from a library. Our tool, qc3C, implements an algorithm that estimates the fraction of reads in the library that contain Hi-C junctions, along with other quality metrics. The algorithm builds upon the observation that proximity ligation events are likely to create k-mers that would not naturally occur in the sample. The algorithm uses an empirical cumulative distribution of k-mer depths to compute the probability that a read containing a Hi-C junction sequence was generated by the proximity ligation reaction. This in turn enables the total fraction of reads containing Hi-C junctions to be estimated. We characterise the accuracy of the new algorithm on simulated and real datasets and compare it to reference-based methods.<br/><br/>qc3C enables sequencing depth requirements to be estimated more precisely on a per-library and per-experiment basis, for chromosome conformation studies, for Hi-C scaffolding of assemblies, and for metagenomic Hi-C. Our qc3C software is an easy to use open-source tool that integrates with the multiQC framework. To our knowledge, qc3C is the first reference-free Hi-C quality assessment tool, enabling Hi-C to be more easily applied to non-model organisms and environmental samples.<br/><br/>qc3C is available from https://github.com/cerebis/qc3C <br/>
