---
layout: post
title:  "A high-quality reference genome for the plant pathogen, Phytophthora cinnamomi"
comments: true
category: plants_fungi
description: "<b>Amy Longmuir, Peter Beech, Mark Richardson</b><br/>The plant pathogen, Phytophthora cinnamomi results..."
videoID: asdf
optimized_image: /assets/img/x2yM7LcXdCSi0bm_title.jpg
tags:
 - pathogens
 - long reads
 - plants
---
{% include cloudstorplayer.html id=page.videoID %}
<u>Amy Longmuir</u><sup>0</sup>, Peter Beech<sup>0</sup>, Mark Richardson<sup>0</sup><br/>
\(0\) Deakin University

Find me during our live conference, [in Remo, table 64](https://remo.co)

<b>Abstract</b><br/>
The plant pathogen, Phytophthora cinnamomi results in noteworthy crop losses globally, including the avocado and macadamia industries in Australia, while also posing a significant threat to the biodiversity of Australiaâ€™s native flora. To date, attempts to sequence and assemble the genome of P. cinnamomi using first and second generation sequencing techniques have resulted in hugely fragmented genomes, with large gaps of missing information. Specifically, significant sections of the genome containing abundant repeat regions and virulence genes are not adequately resolved, limiting the utility of such genomic resources. Utilising recent advances in long-read sequencing by Oxford Nanopore Technologies we sequenced ~5.5 gigabases of an Australian P. cinnamomi isolate \(~50x coverage\). By optimising the assembly process to include error-correction and repeat graphs we have produced the first genome assembly under 1,000 contigs for any Phytophthora sp.; a 85 Mb assembly in 597 contigs with an N50 of 322,750. The presented genome is a vast improvement on the current available reference genome which consists of 9,537 contigs in 1,314 scaffolds, but importantly enables us to understand the importance of large scale structural variations in the genome. Here, we present the results of our assembly and annotation work, and highlight how this improved contiguous assembly enables us to investigate and answer fundamental evolutionary questions regards Phytophthora genomics and pathogenicity. I will then go on to detail our future work \(enabled by this assembly\) where we use whole-genome sequencing to study the mechanisms and evolution of P. cinnamomi infection in important native flora.
