---
layout: post
title:  "3DFAACTS-SNP: Using regulatory T cell-specific epigenomics data to uncover candidate mechanisms of Type-1 Diabetes (T1D) risk"
comments: true
category: regulation
description: "<b>Ning Liu, Timothy Sadlon, Ying Ying Wong, Stephen Pederson, Simon Barry, Jimmy Breen</b><br/>Background<br/>Genome-wide association studies (GW..."
videoID: https://cloudstor.aarnet.edu.au/plus/s/3gKyAoJnCkX0CIj/download
optimized_image: https://cloudstor.aarnet.edu.au/plus/s/ial5U8A7aDMKrre/download
tags:
 - Hi-C
 - transcription
 - GWAS
 - gene function
---
{% include cloudstorplayer.html id=page.videoID %}
<u>Ning Liu</u><sup>0</sup>, Timothy Sadlon<sup>1</sup>, Ying Ying Wong<sup>1</sup>, Stephen Pederson<sup>2</sup>, Simon Barry<sup>1</sup>, Jimmy Breen<sup>0</sup><br/>
\(0\) South Australian Health and Medical Research Institute, University of Adelaide<br/>
\(1\) Women's and Children's Hospital Adelaide, University of Adelaide<br/>
\(2\) University of Adelaide

Find me during our live conference, [in Remo, table 79](https://remo.co)

<b>Abstract</b><br/>
Background<br/>Genome-wide association studies \(GWAS\) have enabled the discovery of single nucleotide polymorphisms \(SNPs\) that are significantly associated with many autoimmune diseases including type 1 diabetes \(T1D\). However, many of the identified variants lie in non-coding regions, limiting the identification of mechanisms that contribute to autoimmune disease progression. To address this problem, we developed a variant filtering workflow called 3DFAACTS-SNP using cell type-specific data integration to link genetic variants that are associated with T1D to the loss of immune tolerance in regulatory T cells \(Treg\).<br/>Results<br/>Using 3DFAACTS-SNP we identified 36 SNPs with plausible Treg-specific mechanisms of action contributing to T1D from 1,228 T1D fine-mapped variants, identifying 119 novel interacting regions resulting in the identification of 51 candidate target genes. We further demonstrated the utility of the workflow by applying it to three other meta-analysed SNP autoimmune datasets, identifying 17 Treg-centric candidate variants and 35 interacting genes. Finally, we demonstrate the broad utility of 3DFAACTS-SNP for functional annotation of all known common \(&gt;10% allele frequency\) variants from the Genome Aggregation Database \(gnomAD\). We identified 7,900 candidate variants and 3,245 candidate target genes, generating a list of potential sites for future T1D or autoimmune research. <br/>Conclusions<br/>We demonstrate that it is possible to further prioritise variants that contribute to T1D based on regulatory function and illustrate the power of using cell type specific multi-omics datasets to determine disease mechanisms. Our workflow can be customised to any cell type for which the individual datasets for functional annotation have been generated, giving broad applicability and utility.
