---
layout: post
title:  "New Insights Of Cancer DNA Methylation By Studies Of Pre-Leukemic Stem Cells In A Mouse Model Of T-Cell Acute Lymphoblastic Leukemia"
comments: true
category: biomed_informatics
description: "<b>Feng Yan, David J. Curtis, Francine E. Garrett-Bakelman, Nicholas C. Wong, Pieter Van Vlierberghe</b><br/>The role of DNA methylation in the initiation and ..."
videoID: https://cloudstor.aarnet.edu.au/plus/s/ykPd1DpegdOdpiR/download
optimized_image: https://cloudstor.aarnet.edu.au/plus/s/ceocnY3a6nzjjN3/download
tags:
 - DNA methylation
 - cancer
 - disease
 - biomedical informatics
 - evolution
---
{% include cloudstorplayer.html id=page.videoID %}
<u>Feng Yan</u><sup>0</sup>, David J. Curtis<sup>0</sup>, Francine E. Garrett-Bakelman<sup>1</sup>, Nicholas C. Wong<sup>0</sup>, Pieter Van Vlierberghe<sup>2</sup><br/>
\(0\) Monash University<br/>
\(1\) University of Virginia<br/>
\(2\) Ghent University

Find me during our live conference, [in Remo, table 54](https://remo.co)

<b>Abstract</b><br/>
The role of DNA methylation in the initiation and clonal evolution of cancer remains poorly understood, in part due to lack of studies of the early pre-malignant state. Recent studies showed that variably methylated regions are associated with multiple cancers, but how it regulates gene expression remains unknown due to sample heterogeneity. To address this, we have analysed three stages of leukemogenesis using a Lmo2 transgenic mouse model of T-cell acute lymphoblastic leukemia \(T-ALL\). FACS purified pre-leukemic stem cells \(Pre-LSCs\), LSCs, bulk T-ALL and wild-type controls were profiled with enhanced reduced representation bisulfite sequencing \(ERRBS\) for DNA methylation and RNA-seq for gene expression.  <br/><br/>Hierarchical clustering for DNA methylation showed the greatest change occurred between pre-LSCs to LSCs. Hypermethylation predominated in pre-LSCs and LSCs, with hypomethylation predominantly in T-ALL. The genomic location of differentially methylations cytosines \(DMCs, compared with earlier stages\) in pre-LSCs and LSCs were distinct. In pre-LSCs, DMCs occurred most frequently in CpG Open Seas and ChIP-Atlas analysis showed enrichment for histone marks of active enhancers \(H3K27Ac/H3K4me2\), the demethylase Kdm1a and the transcription factor RUNX1. In contrast, DMCs in LSCs were more typical of those reported in multiple cancers and ageing, which were most frequently seen at GpG islands and ChIP-Atlas analysis of these sites identified enrichment for the repressive H3K27me3 and activating H3K4me2 modifications. Finally, the transition from LSCs to T-ALL saw new regions of hypomethylated DMCs in CpG open sea.<br/><br/>Integrative analysis of ERRBS and RNA-seq data showed that the differentially methylated regions \(DMRs\) in pre-LSCs were not associated with altered gene expression. However, hypermethylated promoter DMRs of LSCs were correlated with down regulation of 201 genes, including multiple transcription factors, growth factors and signal transduction molecules. Although these genes were lowly expressed in wild-type and pre-LSCs compared to background genes, they were further down regulated in LSCs.<br/><br/>Because pre-LSCs are the earliest stage of TALL development, we asked whether there are already changes of methylation at a clonal level. We calculated an epi-polymorphism score, measured by the heterogeneity of 4 adjacent CpG sites \(epialleles\). The epi-polymorphism increased in pre-LSCs and further in LSCs but decreased in T-ALL. Most differentially heterogenous epialleles \(DHE\) did not overlap with DMRs at the same stage, but half of all DHEs in pre-LSCs was sites of DMRs in LSCs, suggesting that heterogeneity of DNA methylation is a pre-seeding event. Pathways analysis of LSC-specific DHEs showed enrichment for Wnt signaling and pluripotency pathways, exemplified by Wnt3a and Wnt10a.<br/><br/>In conclusion, we have used mouse model of T-ALL to describe the DNA methylation and associated gene expression changes in leukemic stem cells during leukemogenesis. We show for the first time that the well-recognised promoter hypermethylation at bivalently marked sites is preceded by clonal heterogeneity and hypermethylation of Open Seas at active enhancer regions. We propose that these early changes establish a platform for clonal selection of gene expression changes that promote leukemogenesis. Further study of these early changes will provide new insights into the mechanism and role of DNA methylation in cancer development.<br/>
