---
layout: post
title:  "Sex-specific co-expression: a baseline to explore disease"
comments: true
category: single_cell
description: "<b>Sara Ballouz</b><br/>Biological states such as cell-type, cell-state, t..."
videoID: asdf
optimized_image: /assets/img/x2yM7LcXdCSi0bm_title.jpg
invited_talk: 1
tags:
 - single cell
 - transcriptomics
 - networks
 - sex differences
---
{% include cloudstorplayer.html id=page.videoID %}
<u>Sara Ballouz</u><sup>0</sup><br/>
\(0\) Garvan Institute of Medical Research


<b>Abstract</b><br/>
Biological states such as cell-type, cell-state, tissue, sex, disease, or age, are all encoded in a sampleâ€™s transcriptional profile, as measured by assaying mRNA levels. Transcriptomic data has therefore been used as proxy for functional phenotypes. Yet, no gene acts alone, with expression levels of one gene influencing the expression levels of others. Co-variation of gene expression profiles \(i.e., co-expression\) is used to extract information on co-transcription, co-regulation and co-functionality. Therefore, we can also expect that sex differences in the structure of gene co-expression networks can generate sexual dimorphism in downstream phenotypes. However, this still remains relatively unknown. Here, we explore and characterize sex- and tissue- specific derived co-expression networks in order to provide a baseline for disease specific applications. We find very subtle differences between the sexes, suggesting that sex differences in expression may be relevant to very specific functions and pathways, necessitating careful exploration.
