---
layout: post
title:  "Optimising genomic approaches for detection of vancomycin resistant Enterococcus faecium transmission in the hospital environment"
comments: true
category: methods
description: "<b>Charlie Higgs, Claire Gorrie, Norelle Sherry, Ben Howden</b><br/>Objectives:<br/><br/>Vancomycin-resistant Enteroco..."
videoID: asdf
optimized_image: /assets/img/x2yM7LcXdCSi0bm_title.jpg
tags:
 - genomics
 - epidemiology
 - resistance
 - disease
---
{% include cloudstorplayer.html id=page.videoID %}
<u>Charlie Higgs</u><sup>0</sup>, Claire Gorrie<sup>1</sup>, Norelle Sherry<sup>1</sup>, Ben Howden<sup>1</sup><br/>
\(0\) University of Melbourne<br/>
\(1\) Microbiological Diagnostic Unit Public Health Laboratory

Find me during our live conference, [in Remo Room 2, table 35](https://remo.co)

<b>Abstract</b><br/>
Objectives:<br/><br/>Vancomycin-resistant Enterococcus faecium \(VREfm\) is a leading cause of nosocomial infections and globally significant public health threat. In an effort to better understand the transmission of VREfm within the hospital environment, whole genome sequencing \(WGS\) is increasingly being used. While the flexibility and plasticity of the VREfm genome enables its adaptation to the hospital environment, it limits our ability to identify transmission events. Currently multiple genomic analysis approaches are being used worldwide without consensus. This project aims to determine the optimal method for analysing WGS data to identify transmission events, using epidemiological data for comparison. The optimal method would be stable over time as new isolates are added to the analysis, be standardised to allow for comparison across sites and require minimal interpretation.<br/><br/>Methods:<br/><br/>This study combined WGS data from VREfm samples \(n= 305\) with comprehensive patient bed movement data, collected during a 15-month prospective study across four Victorian hospital networks \(Controlling Superbugs study\). To determine the most reliable method for identifying possible transmission events, multiple genetic comparison methods were used, including: core genome single nucleotide polymorphism \(SNP\) count grouped by multi locus sequence type \(MLST\), core genome MLST \(cgMLST\), k-mer based comparison method \(SKA\) and pairwise mapping following de novo assembly. This was compared with the patients’ bed movement data to infer likelihood of transmission based on temporal and spatial overlap. Current literature suggests an isolate pairwise distance threshold of ≤25 SNPs indicates putative transmission of VREfm. <br/><br/>Results:<br/><br/>Genomic comparison methods that compare isolates in a pairwise manner \(SKA and de novo pairwise comparison\) were stable over time and can easily be standardised. When set at the same relatedness thresholds, cgMLST displayed a lower concordance with the epidemiological data compared to the core genome alignments but could be more easily standardised. Methods that maximise the amount of genomic diversity captured \(SKA and de novo pairwise comparison\) showed greater discriminatory power in identifying transmission clusters that were supported by ward-move data. Nested clusters were able to be separated within groups of isolates that were indistinguishable using core genome alignment methods. We also showed that although a SNP threshold of ≤25 may be effective for some data sets, it cannot be uniformly applied across all ST backgrounds using core genome alignments.  <br/><br/>Conclusion:<br/><br/>Of the genomic comparison methods tested, those that maximise the genomic diversity captured, such as SKA and the de novo pairwise comparison, showed better concordance with the epidemiological data. These methods could increase the sensitivity of VREfm transmission analyses and more discriminately identify outbreaks. <br/>
