---
layout: post
title:  "Detection and identification of cis-regulatory elements using change-point and classification algorithms"
comments: true
category: regulation
description: "<b>Dominic Maderazo, Jennifer Flegg, Manjula Algama, Mirana Ramialison, Jonathan Keith</b><br/>Transcriptional regulation is primarily mediated b..."
videoID: asdf
optimized_image: assets/img/x2yM7LcXdCSi0bm_title.jpg
tags:
 - sequence analysis
 - sequence alignment
 - classification algorithms 
---
{% include cloudstorplayer.html id=page.videoID %}
<u>Dominic Maderazo</u><sup>0</sup>, Jennifer Flegg<sup>0</sup>, Manjula Algama<sup>1</sup>, Mirana Ramialison<sup>2</sup>, Jonathan Keith<sup>3</sup><br/>
\(0\) The University of Melbourne<br/>
\(1\) RMIT<br/>
\(2\) The Australian Regenerative Medicine Instute<br/>
\(3\) Monash University

Find me during our live conference, [in Remo Room 2, table 30](https://remo.co)

<b>Abstract</b><br/>
Transcriptional regulation is primarily mediated by the binding of factors to non-coding regions in DNA. Identification of these binding regions enhances understanding of tissue formation and potentially facilitates the development of gene therapies. However, successful identification of binding regions is made difficult by the lack of a universal biological code for their characterisation. We extend an alignment-based method, changept, and identify clusters of biological significance, through ontology and de novo motif analysis. Further, we apply a Bayesian method to estimate and combine binary classifiers on the clusters we identify to produce a better performing composite. The analysis we describe provides a computational method for identification of conserved binding sites.
