---
layout: post
title:  "Developing a computational analysis to identify differentially allelic expressed loci in patient-derived stem cells"
comments: true
category: methods
description: "<b>Daniel Russell, Alexandre S. Cristino</b><br/>Epigenetics is the inherited and acquired modifica..."
videoID: https://cloudstor.aarnet.edu.au/plus/s/3qvCIKtznkAmMpp/download
optimized_image: https://cloudstor.aarnet.edu.au/plus/s/0Hnc1bV8kHPMgIL/download
tags:
 - epigenomics
 - transcriptomics
 - disease
 - DNA sequencing
---
{% include cloudstorplayer.html id=page.videoID %}
<u>Daniel Russell</u><sup>0</sup>, Alexandre S. Cristino<sup>0</sup><br/>
\(0\) Griffith Institute for Drug Discovery

Find me on Wed Nov 25th, 1:30-2:50pm AEDT [in Remo, table 124](https://live.remo.co/e/abacbs2020-day-2/register)

<b>Abstract</b><br/>
Epigenetics is the inherited and acquired modifications in the genome that affect the gene activity without changing the DNA sequence. Instead the epigenome is determined by chemical modifications within the genome such as the modification of histone proteins and the methylation of DNA which in turn change gene regulation. Epigenetics plays a vital role in the modulation of gene activity, especially during embryonic development and in brain function. Recent research has found that it also plays a role in disease, particularly neurodegenerative and neuropsychiatric diseases such as Alzheimerâ€™s disease and schizophrenia. An intriguing type of epigenetic regulation is genomic imprinting, where one of the two alleles of a particular gene are silenced permanently depending on its parent of origin. Other instances of allele-specific expression are of great interest as when the expression of a heterozygous gene deviates towards one allele, it becomes functionally homozygous. In the case of risk alleles, this level of deviation may increase its impact in the onset of polygenic diseases. <br/>This study aims to develop a computation analysis to identify protein-coding genes with allele-specific expression by integrating whole genome sequencing \(WGS\) data and RNA sequencing \(RNA-seq\) data from the same patient-derived stem cells.  We implemented a computational pipeline to determine differential allele expression based on read counts from RNA-Seq data of all heterozygous loci identified in WGS data from the same patient-derived stem cells. Heterozygous variants that showed statistically significant biased expression were validated using a gold standard data set for known imprinted genes and available epigenome maps. Our pipeline may be useful for finding epigenetic regulated loci associated with diseases using patient-derived stem cells. As a proof-of-concept we used olfactory neuronal stem cells from a well-studied schizophrenia cohort. Many of these epigenetically regulated loci have been previously identified to be associated with schizophrenia. We propose heterozygous risk alleles located in these loci may have very different ranges of functional effects based on their allelic-biased expression which is defined by epigenomic regulation.
