---
layout: post
title:  "Binning Metagenomic Sequences"
comments: true
category: genomics
description: "<b>Yu Lin</b><br/>Metagenomics studies have provided key insights in..."
videoID: https://cloudstor.aarnet.edu.au/plus/s/dF6DZb5MlZvXUVq/download
optimized_image: https://cloudstor.aarnet.edu.au/plus/s/4m6IesQpPn4QBK5/download
invited_talk: 1
tags:
 - metagenomics
 - genome assembly
 - graph algorithms
---
{% include cloudstorplayer.html id=page.videoID %}
[<u>Yu Lin</u>](http://users.cecs.anu.edu.au/~u1024708/)<sup>0</sup><br/>
\(0\) Research School of Computer Science, Australian National University


<b>Abstract</b><br/>
Metagenomics studies have provided key insights into the composition and structure of microbial communities found in different environments. Among the techniques used to analyze metagenomic data, binning is considered as a crucial step in order to characterize the different species of microorganisms present in microbial communities. We propose two binning methods, GraphBin and MetaBCC-LR, to bin metagenomic sequences. GraphBin makes use of the assembly graph to refine binning results and to enable detecting shared sequences among multiple species. MetaBCC-LR is a new reference-free binning method which directly clusters long reads based on their k-mer coverage histograms and oligonucleotide composition.
