---
layout: post
title:  "Developing a computational safeguard to detect gene drive systems in wild populations"
comments: true
category: nonmodel
description: "<b>Aidan Tay, Cameron Hosking, Brendan Hosking, Suzanne Scott, Daniel Reti, Aidan O'Brien, Denis Bauer, Laurence Wilson</b><br/>Genome editing technologies such as CRISPR-Cas9, h..."
videoID: asdf
optimized_image: /assets/img/x2yM7LcXdCSi0bm_title.jpg
session_talk: 1
tags:
 - genomics
 - genome editing
 - biosecurity
 - non-model organisms
---
{% include cloudstorplayer.html id=page.videoID %}
<u>Aidan Tay</u><sup>0</sup>, Cameron Hosking<sup>0</sup>, Brendan Hosking<sup>0</sup>, Suzanne Scott<sup>0</sup>, Daniel Reti<sup>0</sup>, Aidan O'Brien<sup>0</sup>, Denis Bauer<sup>0</sup>, Laurence Wilson<sup>0</sup><br/>
\(0\) CSIRO

Find me during our live conference, [in Remo Room 2, table 7](https://remo.co)

<b>Abstract</b><br/>
Genome editing technologies such as CRISPR-Cas9, have made it possible to engineer gene drive systems for spreading desirable traits throughout wild populations. These systems rely on the fact that some genetic elements have a higher chance of being inherited, thereby allowing them to ‘drive’ through a population over many generations. By releasing genetically modified individuals containing gene drive systems and allowing them to breed with wild individuals, desirable traits for managing wild populations such as invasive species or disease vectors, can be propagated. However, the use of gene drive systems for managing wild populations remains hampered by the potential risks associated with releasing genetically modified individuals containing these systems. To help minimise the risks associated with releasing genetically modified individuals into the wild and improve the traceability of gene drive systems, we developed a computational approach for detecting the presence of gene drive systems within a genome. This is done by analysing the characteristic frequency of oligonucleotide sequences \(i.e., genomic signature\). Different organisms display unique genetic signatures, which can be used to differentiate DNA originating from different species. By analysing the changes in genomic content of sequences at different locations, the native DNA sequence and that of a gene drive can be distinguished. We demonstrate how gene drive systems can be detected in whole genome sequencing data derived from experimental sequencing library for yeast, and a theoretical sequencing library for the Cas9 gene. Importantly, this approach requires no prior knowledge about the genomic sequence and requires no alignment to a reference sequence, meaning it can be readily applied to poorly characterized organisms.
