---
layout: post
title:  "The AMPSphere: antimicrobial peptides (AMPs) in the global microbiome"
comments: true
category: metagenomics
description: "<b>Célio Dias Santos-Júnior, Thomas S. B. Schmidt, Anthony Fullam, Peer Bork, Xing-Ming Zhao, Luis Pedro Coelho</b><br/>Antimicrobial peptides (AMPs) are short peptides t..."
videoID: asdf
optimized_image: /assets/img/x2yM7LcXdCSi0bm_title.jpg
session_talk: 1
tags:
 - metagenomics
 - microbiome
 - antimicrobial peptides
---
{% include cloudstorplayer.html id=page.videoID %}
Célio Dias Santos-Júnior<sup>0</sup>, Thomas S. B. Schmidt<sup>1</sup>, Anthony Fullam<sup>1</sup>, Peer Bork<sup>2</sup>, Xing-Ming Zhao<sup>3</sup>, <u>Luis Pedro Coelho</u><sup>0</sup><br/>
\(0\) Fudan University<br/>
\(1\) European Molecular Biology Laboratory<br/>
\(2\) embl<br/>
\(3\) Tongji University

Find me during our live conference, [in Remo Room 1, table 29](https://remo.co)

<b>Abstract</b><br/>
Antimicrobial peptides \(AMPs\) are short peptides that inhibit the growth of microbial organisms Particularly as antimicrobial resistance becomes a public-health crisis, AMPs can be valuable for use in both clinical and industrial applications. At the same time, it is important to recall that these molecules did not evolve for human benefit, but they must play a role in the natural habitats and we expect that they have an impact in structuring microbial communities. Due to their small size, standard gene mining approaches are not directly applicable to short peptides. To address this problem, we previously developed macrel, a machine-learning based pipeline to find AMPs in \(meta\)genomes.<br/><br/>Having validated macrel on benchmark and test datasets, we then applied it to ProGenomes2, which includes 86 thousand high-quality genomes, as well as a dataset of over 35,000 publicly available metagenomes. After redundancy removal, we produced a collection of AMPs from the global microbiome, which we termed the AMPsphere.<br/><br/>This first version of the AMPsphere contains 317,790 distinct amino acid sequences \(97.1% of which were found exclusively in metagenomes\). These sequences were further clustered by sequence similarity into 4,705 AMP families by sequence similarity.<br/><br/>While macrel finds peptides that are present as short genes \(as opposed to being formed by fragmentation of larger proteins\), 9% of the AMPsphere peptides could be matched to larger proteins in the eggNOG databases. These can represent cryptic peptides that originated as fragments and have evolved to become independent genes.<br/><br/>Current work aims at elucidating the impact of AMPs on microbial communities and quantifying the extent to which they play a role in structuring the community.<br/>
