---
layout: post
title:  "RNase III-CLASH of multi-drug resistant Staphylococcus aureus reveals a regulatory mRNA 3’UTR required for intermediate vancomycin resistance"
comments: true
category: transcriptomics
description: "<b>Daniel G. Mediati, Julia L. Wong, Wei Gao, Stuart MacKellar, Chi Nam Ignatius Pang, Sylvania Wu, Winton Wu, Brandon Sy, Ian Monk, Joanna Richmond, Benjamin Howden, Tim P. Stinear, Sander Granneman, Jai J. Tree</b><br/>Treatment of methicillin-resistant Staphylococcus ..."
videoID: asdf
optimized_image: assets/img/x2yM7LcXdCSi0bm_title.jpg
tags:
 - RNA-RNA interactions
 - networks
 - antibiotic resistance
 - transcriptomics
 - bacteria
---
{% include cloudstorplayer.html id=page.videoID %}
Daniel G. Mediati<sup>0</sup>, Julia L. Wong<sup>0</sup>, Wei Gao<sup>1</sup>, Stuart MacKellar<sup>2</sup>, [<u>Chi Nam Ignatius Pang</u>](http://www.babs.unsw.edu.au/staff_directory/wilkinsm.html)<sup>0</sup>, Sylvania Wu<sup>0</sup>, Winton Wu<sup>0</sup>, Brandon Sy<sup>0</sup>, Ian Monk<sup>1</sup>, Joanna Richmond<sup>3</sup>, Benjamin Howden<sup>1</sup>, Tim P. Stinear<sup>1</sup>, Sander Granneman<sup>2</sup>, Jai J. Tree<sup>0</sup><br/>
\(0\) School of Biotechnology and Biomolecular Sciences, University of New South Wales, Sydney, NSW Australia<br/>
\(1\) Department of Microbiology and Immunology, Peter Doherty Institute, University of Melbourne, Melbourne VIC Australia<br/>
\(2\) Centre for Systems and Synthetic Biology, University of Edinburgh, Edinburgh, United Kingdom<br/>
\(3\) Electron Microscopy Unit, University of New South Wales, Kensington, NSW Australia

Find me during our live conference, [in Remo Room 1, table 39](https://remo.co)

<b>Abstract</b><br/>
Treatment of methicillin-resistant Staphylococcus aureus \(MRSA\) infections is dependant on the efficacy of last-line antibiotics like vancomycin. Vancomycin treatment failure is most commonly linked to the emergence of vancomycin-intermediate resistance in clinical isolates \(termed VISA\). These isolates have not acquired resistance genes but appear to accumulate a heterogenous collection of single nucleotide polymorphism that collectively alter the physiology of the cell to increase vancomycin tolerance. Cell wall thickening is common among VISA isolates and is thought to decrease vancomycin permeability. Changes in regulatory sRNA expression have been correlated with antibiotic stress responses in VISA isolates however the functions of the vast majority of these RNA regulators is unknown. The 5’ and 3’ untranslated regions \(UTRs\) of mRNAs are often the site of regulatory RNA interactions. Therefore, we generated a highly detailed transcriptome architecture of methicillin-resistant Staphylococcus aureus JKD6009. This include using RNA-seq to identify expressed transcripts, differential RNA-sequencing \(Sharma and Vogel, 2014, Curr Opin in Microbiol, 19:97–105\) to identify transcription start sites, and the use of Term-Seq to identify transcripts termination site \(Dar et al., 2016, Science, 352:aad9822\). The ANNOgesic pipeline \(Yu et al., 2018, GigaScience, 7\(9\):giy096\) was used to analyse the transcriptome data to generate the detailed transcriptome map, which resulted in the identification of 2867 coding sequences, 1156 5' UTRs, 1031 3' UTRs, and 499 sRNAs. We have used the endoribonuclease RNase III to capture RNA-RNA interactions using an RNA proximity-dependant ligation technique termed CLASH. From seven independent RNase III-CLASH experiments, 256 sRNA-mRNA interactions were observed in vivo allowing functional characterisation of many sRNAs for the first time. Surprisingly, we found that an mRNA encoding an unusually long 3’UTR \(here termed vigR\) functions as a regulatory ‘hub’ within our RNA-RNA interaction network. We present evidence that vigR promotes expression of the cell wall lytic transglycosylase encoded by isaA through a direct mRNA-mRNA interaction. Further, we found that the vigR mRNA 3’UTR is required for cell wall thickening and that deletion of the vigR 3’UTR re-sensitises VISA to vancomycin. Our results demonstrate the utility of RNase III-CLASH for identifying new regulatory RNA functions and indicate that S. aureus may use mRNA-mRNA interactions to co-ordinate gene expression much more widely than previously appreciated. 
