---
layout: post
title:  "A machine learning model to predict splice factor expression directly from transcriptome-wide splicing patterns  "
comments: true
category: transcriptomics
description: "<b>A.J. Sethi, Pablo Acera Mateos, Renzo Balboa, Emiliana Weiss, Attila Horvath</b><br/>Dysregulated splicing is a major driver of cancer ..."
videoID: https://cloudstor.aarnet.edu.au/plus/s/47jI0Mx1W6TV8Ml/download
optimized_image: https://cloudstor.aarnet.edu.au/plus/s/orODGJDw8GxrXg5/download
tags:
 - RNA splicing
 - networks
 - gene regulation
 - machine learning
---
{% include cloudstorplayer.html id=page.videoID %}
<u>A.J. Sethi</u><sup>0</sup>, Pablo Acera Mateos<sup>0</sup>, Renzo Balboa<sup>0</sup>, Emiliana Weiss<sup>0</sup>, Attila Horvath<sup>0</sup><br/>
\(0\) John Curtin School of Medical Research

Find me on Tues Nov 24th, 1:40-3pm AEDT [in Remo, table 131](https://live.remo.co/e/abacbs2020-day-1/register)

<b>Abstract</b><br/>
Dysregulated splicing is a major driver of cancer and inherited genetic disease, of which the underlying mechanisms are poorly understood. Variation in splicing outcomes \(isoform usage\) can arise from several sources, including the differential expression of splice factors \(SF\). These proteins regulate spliceosome assembly, and may function both synergistically and redundantly to coordinate splice-site selection. Although numerous studies have characterised transcriptome-wide differential splicing patterns following the depletion or overexpression of individual SFs, the observed differential splicing phenotypes are always a cumulative effect of the experimental treatment in combination with the compensatory expression of other, interdependently-regulated splice factors. As such, these conventional methods fail to link individual splice factors directly to the alternative splicing events which they regulate. <br/><br/>Our study aims to quantify the impact of the siRNA depletion of 56 individual splicing-related proteins on the global expression of splice factors, and to further to measure the resultant alternative splicing patterns in a publicly available dataset in drosophila \(Brooks et al. 2015\). Using this data, we aim to develop a statistical learning model to understand the complex relationships between transcriptome-wide splicing patterns \(i.e. exon percentage spliced-in\) and the underlying splice-factor transcript expression levels. This model will allow for the prediction of SF expression levels directly from exon inclusion values. Using this information, we will gain further insight into the roles of individual SF in controlling alternative splicing. Furthermore, we will gain insight in how we may be able to modulate disease-relevant alternative splicing evets directly at the splice factor level. In contrast to previous approaches to understand the roles of splice factors, our model will directly quantify the proportion of variation in isoform usage that is driven by differential splice factor expression, clearly delineating the effects of SF from the other mechanisms which regulate pre-mRNA splicing.<br/><br/>References: <br/><br/>Brooks AN, Duff MO, May G, et al. Regulation of alternative splicing in Drosophila by 56 RNA binding proteins. Genome Res. 2015;25\(11\):1771-1780. doi:10.1101/gr.192518.115<br/><br/><br/>â€ƒ<br/>
