---
layout: post
title:  "Finding signatures of alternative polyadenylation as cancer biomarkers"
comments: true
category: biomed_informatics
description: "<b>Nitika Kandhari, Paul Harrison, Kaylene Simpson, David Powell, Traude Beilharz, Iva Nikolic, Robert Vary, Karla Cowley</b><br/>Alternative transcript cleavage and Polyadenylatio..."
videoID: asdf
optimized_image: /assets/img/x2yM7LcXdCSi0bm_title.jpg
session_talk: 1
tags:
 - transcriptomics
 - cancer
 - polyadenylation
---
{% include cloudstorplayer.html id=page.videoID %}
<u>Nitika Kandhari</u><sup>0</sup>, Paul Harrison<sup>0</sup>, Kaylene Simpson<sup>1</sup>, David Powell<sup>0</sup>, Traude Beilharz<sup>0</sup>, Iva Nikolic<sup>0</sup>, Robert Vary<sup>1</sup>, Karla Cowley<sup>1</sup><br/>
\(0\) Monash University<br/>
\(1\) The University of Melbourne

Find me during our live conference, [in Remo Room 1, table 22](https://remo.co)

<b>Abstract</b><br/>
Alternative transcript cleavage and Polyadenylation \(APA\) is linked with cancer cell transformation and proliferation. APA gene biomarkers are strong predictors of clinical outcomes. If incorporated to standard prognostic measures such as gene-expression and clinical parameters, these could reform cancer prognostic testing and therapy. Nearly 70% of mammalian genes harbour APA sites resulting in distinct transcripts with variable 3’UTR length and regulatory content. Short 3’UTRs are generally associated with de-differentiated proliferative cells \(eg, stem cells\) whereas longer 3’-UTRs associate with more complex regulation and cellular specialisation. ~91% APA genes switch to shorter mRNA isoforms in tumour cells. Our study aims to detect signatures of APA changes that are specific to triple-negative breast cancer \(TNBC\) that could be applied as a novel prognostic biomarker in early-stage breast cancer. Using bioinformatic analyses of 3’-focused RNA-seq approaches we studied the landscape of transcription and APA in three cancer cell lines in response to loss of PCF11, a core regulator of 3’-end formation. This shows a conservation of an expression and processing response to loss of 3’-end processing machinery. In addition to gene expression changes, we identify 3’-end “shifted” signature genes that are common to all 3 cell lines. These signature genes highlight a different biology than gene expression. We will present our current work around the idea that systematic lengthening of 3’UTR interferes with key signalling pathways and sensitizes cells to PTEN/AKT inhibitor.
