---
layout: post
title:  "New insights into plant-microbe interactions through Quantitative Trait Locus (QTL) mapping"
comments: true
category: plants_fungi
description: "<b>Charlotte Francois</b><br/>Like humans, plants are colonised by bacteria on v..."
videoID: asdf
optimized_image: assets/img/x2yM7LcXdCSi0bm_title.jpg
session_talk: 1
tags:
 - microbiome
 - plants
 - QTL
 - sequence variation
 - statistics
 - bacteria
---
{% include cloudstorplayer.html id=page.videoID %}
[<u>Charlotte Francois</u>](https://www.linkedin.com/in/charlotte-francois/)<sup>0</sup><br/>
\(0\) La Trobe University

Find me during our live conference, [in Remo Room 1, table 13](https://remo.co)

<b>Abstract</b><br/>
Like humans, plants are colonised by bacteria on virtually every surface. Surrounding roots is a carefully-regulated region known as the rhizosphere, which supports a special subset of bacteria known as beneficial rhizobacteria. Plants regulate rhizosphere structure and function through the secretion of root exudates that contain sugars, secondary metabolites and phenolics suspended in an exopolysaccharide mucilage. The activity and makeup of the rhizosphere-associated bacterial community is similarly regulated by the plant host. Beneficial rhizobacteria provide numerous benefits to their hosts, including but not limited to improved nutritional status and enhanced stress and pathogen tolerance. In exchange, beneficial bacteria receive sugars and other metabolites. <br/><br/>This ancient relationship is representative of a long coevolutionary process and selective pressure on both partners has been applied. The magnitude of the benefit conferred is partially dependent on the host plant genotype and many studies have demonstrated that there is genetic variation for this trait. Studies using Arabidopsis thaliana have shown that varieties within a plant species respond differently to a single beneficial rhizobacterial species, which suggests that there is host genetic variation for this trait. It is therefore expected that rhizosphere-associated traits in plants are adaptive traits, and that an enhanced ability to gain from beneficial rhizobacteria may be evolutionarily advantageous, having consequences that affect plant fitness.<br/><br/>The genes underlying this variation are currently unknown, but they represent attractive targets in emerging biotechnologies that seek to exploit the interaction between plants and beneficial bacteria, such as phytoremediation. Using a small Arabidopsis recombinant inbred line \(RIL\) mapping population combined with Quantitative Trait Locus mapping and candidate gene identification in R, a computational genomic analysis was conducted to understand the complex interaction between plants and their beneficial rhizobacteria, elucidate underlying genes and provide new insights into this variation.
