---
layout: post
title:  "Viral RNA metagenomics reveals the Australian bovine respiratory virome"
comments: true
category: transcriptomics
description: "<b>Barbara Brito Rodriguez, Kay Anantanawat, Tony Batterham, Melinda Frost, Peter Kirkland, Edward Holmes, Aaron Darling</b><br/>Metagenomic next-generation sequencing is transfor..."
videoID: asdf
optimized_image: assets/img/x2yM7LcXdCSi0bm_title.jpg
tags:
 - Virus
 - metagenomics
 - livestock
 - bovine
 - RNA sequencing
 - virome
 - antimicrobial resistance
 - agriculture
---
{% include cloudstorplayer.html id=page.videoID %}
<u>Barbara Brito Rodriguez</u><sup>0</sup>, Kay Anantanawat<sup>1</sup>, Tony Batterham<sup>2</sup>, Melinda Frost<sup>3</sup>, Peter Kirkland<sup>3</sup>, Edward Holmes<sup>4</sup>, Aaron Darling<sup>5</sup><br/>
\(0\) UNIVERSITY OF TECHNOLOGY SYDNEY<br/>
\(1\) University of Technology Sydney<br/>
\(2\) APIAM Animal Health<br/>
\(3\) Elizabeth Macarthur Agricultural Institute, NSW, DPI<br/>
\(4\) The University of Sydney<br/>
\(5\) The University of Technology Sydney

Find me during our live conference, [in Remo Room 1, table 12](https://remo.co)

<b>Abstract</b><br/>
Metagenomic next-generation sequencing is transforming public health and has the potential to do the same with the agricultural industries and national biosecurity. Beef production contributes to 1.5% of Australia’s key industry GDP. The most impactful infectious disease in intensive beef cattle production is bovine respiratory disease \(BRD\). Traditionally a limited number of bacteria and viruses were thought to be associated with respiratory disease. Recent metagenomic studies have unveiled a number of additional viruses in the respiratory tract of bovines, potentially associated with respiratory disease. We collected pharyngeal swabs from two feedlots in NSW to characterise the virome of beef cattle in Australia. The samples were collected in viral transport media and transported on dry ice to the Bioscience laboratory at the University of Technology Sydney. The RNA was extracted using Qiagen RNeasy Micro kit. Samples were pooled to obtain a minimum of 3 ug of RNA. Library preparation was done using the Illumina TruSeq® Stranded  \(human, mouse, rat\) kit for library preparation and sequencing was run in an Illumina NovaSeq S1 Lane, 300 cycles \(150bp paired\) at AGRF. A total of 353 Gb of data was generated from 15 pooled libraries. The Bos taurus genome assembly \(GCA\_002263795.2\) was used to filter out host reads using bwa mem and samtools. The reads were assembled and taxonomically classified using the Genome Detective online platform. Briefly, low-quality reads are filtered out using trimommatic. Then, the reads are classified using DIAMOND and the Swissprot Uniref90 protein database. The reads are sorted into groups \(bins\). Each bin contains reads of one viral species. De novo assembly was done in SPAdes for each bin. The contigs were then classified using NCBI RefSeq virus database. Contigs were joined and consolidated using the Advanced Genome Aligner.  The reads were also queried to investigate the presence of antimicrobial resistance genes from the metatranscriptome obtained using ResFinder. <br/><br/>Using an NGS RNA metagenomics approach we obtained near whole-genome sequences of multiple viruses, including Bovine Nidovirus, Bovine Rhinitis A, Bovine Viral Diarrhea Virus-1, Bovine Betacoronavirus, Enterovirus E, as well as the partial genome of Bovine Ungulate tetraparvovirus 1, and a novel paramyxovirus. We also found smaller contigs \(sequences\) that suggest the presence of segments of Influenza D, Bovine Rhinitis B, Ungulate bocaparvovirus 6, Influenza D, Bovine Respiratory Syncytial virus and Parainfluenza 3. Surprisingly, the most abundant virus in feedlot cattle was Bovine Nidovirus, a virus that was only recently discovered in the US \(2015\) and that has only been reported in two other countries to date. One antimicrobial resistance gene was consistently identified in several pools: Beta-lactam bla-TEM-116. Although none of the viruses found in this study are reportable, they are part of an unexplored respiratory complex in Australian cattle. <br/>
