---
layout: post
title:  "Over 25,000 metagenome assembled genomes reveal the development of the post-weaning pig gut microbial community"
comments: true
category: metagenomics
description: "<b>Daniela Gaio, Matthew Z. DeMaere, Kay Anantanawat, Toni A. Chapman, Steven P. Djordjevic, Aaron E. Darling</b><br/>Building on a new lower-cost metagenome sequencing..."
videoID: https://cloudstor.aarnet.edu.au/plus/s/vmXgdZF00SLtn1F/download
optimized_image: https://cloudstor.aarnet.edu.au/plus/s/b49jQwlmQXP5xtr/download
session_talk: 1
tags:
 - metagenomics
 - microbiome
 - genome assembly
 - agriculture
---
{% include cloudstorplayer.html id=page.videoID %}
<u>Daniela Gaio</u><sup>0</sup>, Matthew Z. DeMaere<sup>1</sup>, Kay Anantanawat<sup>1</sup>, Toni A. Chapman<sup>2</sup>, Steven P. Djordjevic<sup>1</sup>, Aaron E. Darling<sup>1</sup><br/>
\(0\) UTS<br/>
\(1\) University of Technology Sydney<br/>
\(2\) NSW Department of Primary Industries

Find me on Wed Nov 25th, 1:30-2:50pm AEDT [in Remo, table 8](https://live.remo.co/e/abacbs2020-day-2/register)

<b>Abstract</b><br/>
Building on a new lower-cost metagenome sequencing technique developed by us, we have carried out the largest metagenomic analysis of the pig gut microbiome to-date. By processing over 800 time-series samples from 126 porcine hosts, we have generated over 8Tbp of metagenomic sequence data. From this data we reconstructed over 50,000 metagenome-assembled genomes \(MAGs\) of organisms resident in the porcine gut, 26,800 of which were above 70% complete with a &lt;10% contamination, and 12,400 of which were nearly complete genomes. To do so we created co-assemblies for each individual host, pooling all the time-series samples available from each subject, thereby increasing the power to reconstruct genomes from low abundance microbes. We find that the microbiomes of post-weaning piglets appear to follow a highly structured developmental program in the weeks following weaning, and this development is robust to treatments including antibiotics and probiotics. The high resolution we obtained allowed us to identify specific taxonomic and genomic “signatures” characterizing the stages that the piglet gut microbiome shifts through during its development immediately after weaning \(4 weeks\) and up to the ninth week of life. These compositional shifts were also evident in  analysis with an assembly-free phylogenetic profiling technique. Both the assembly-free method and the analysis of MAGs show strong associations between community composition and individual factors such as breed and mother, and can even resolve small differences in age \(down to 3 days\). Finally, by predicting proteins from the metagenomes and mapping them against the CAZy database, we described the carbohydrate repertoire of the post-weaning piglets. We tracked the shifts in abundance of these enzymes across time, and identified the species and higher level taxonomic groups that carry each of these enzymes in their metagenomes.
