---
layout: post
title:  "Whole transcripts in genome assembly, annotation, and assessment: the draft genome assembly of the globally invasive common starling, Sturnus vulgaris"
comments: true
category: nonmodel
description: "<b>Katarina Stuart, Yuanyuan Cheng, Lee Rollins, Richard Edwards</b><br/>Native to the Palearctic, the common starling (Stu..."
videoID: https://cloudstor.aarnet.edu.au/plus/s/RSnJeMzBBxaaGJC/download
optimized_image: https://cloudstor.aarnet.edu.au/plus/s/8TBHnGpSpLlcL77/download
session_talk: 1
tags:
 - genome assembly
 - genomics
 - transcriptomics
 - gene annotation
 - non-model organisms
---
{% include cloudstorplayer.html id=page.videoID %}
<u>Katarina Stuart</u><sup>0</sup>, Yuanyuan Cheng<sup>1</sup>, Lee Rollins<sup>0</sup>, Richard Edwards<sup>0</sup><br/>
\(0\) UNSW<br/>
\(1\) USYD

Find me during our live conference, [in Remo, table 106](https://remo.co)

<b>Abstract</b><br/>
Native to the Palearctic, the common starling \(Sturnus vulgaris\) is a near-globally invasive passerine that has now colonised every continent barring Antarctica. Ecological interest in the species is two-fold â€“ they are considered a conservation risk and crop pest within the invasive ranges, while recent decades have brought with them a worrying decline in starling numbers within historical native ranges. Despite the global interest in this species, there are still fundamental knowledge gaps in our understanding of the genetics and population differences of this species across their native and invasive range. We present the Australian S. vulgaris draft genome and transcriptome to be used as a reference for further investigation into evolutionary characterisation of this ecologically significant species. An initial 10x Genomics linked-read assembly was scaffolded and gap-filled with low coverage nanopore sequencing,  complemented by PacBio Isoseq full-length transcript data. Isoseq data was incorporated into assembly scaffolding, annotation, and assembly assessment to inform workflow decisions. We produced a draft assembly with a scaffold N50 size of 72.5 Mb, and assess this alongside a North American S. vulgaris draft genome, previously assembled from Illumina data. Lastly, we use these different reference genomes, alongside a non-scaffolded version of the Australian S. vulgaris genome to assess how choice of reference genome affects common population genetic downstream analysis using a global whole genome resequencing data set.
